tests/tt_eager/python_api_testing/unit_testing/backward_ops/test_backward_atan2.py

```diff
@@ -26,7 +27,7 @@ def test_bw_atan2(input_shapes, device):
 
     pyt_y = torch.atan2(in_data, other_data)
 
-    tt_output_tensor_on_device = tt_lib.tensor.atan2_bw(grad_tensor, input_tensor, other_tensor)
+    tt_output_tensor_on_device = ttnn.atan2_bw(grad_tensor, input_tensor, other_tensor, ttnn.DRAM_MEMORY_CONFIG)
```

Comment:
do you need to explicitly pass memory config? if not passed it is taken from input tensor, right?

Reply:
we've updated the code